src/fu-util.c

```diff
@@ -977,6 +977,23 @@ fu_util_device_test_filename(FuUtilPrivate *priv,
 	return TRUE;
 }
 
+static gboolean
+fu_util_inhibit(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	const gchar *reason = "not set";
+	g_autofree gchar *inhibit_id = NULL;
+
+	if (g_strv_length(values) > 0)
+		reason = values[1];
+
+	/* inhibit, wait, uninhibit */
```

Comment:
This doesn't make sense to me. It should be two separate commands, or if combined command needs to have the delay as an argument.

Reply:
Agreed, two separate commands makes a lot more sense if we put in place a cancellable GMainLoop. Fixed.